src/libs/ReportActionsUtils.ts

```diff
@@ -609,6 +622,60 @@ function isConsecutiveActionMadeByPreviousActor(reportActions: ReportAction[] |
     return currentAction.actorAccountID === previousAction.actorAccountID;
 }
 
+// Todo - combine this with `isConsecutiveActionMadeByPreviousActor` so as to not duplicate logic
+function hasNextActionMadeBySameActor(reportActions: ReportAction[], actionIndex: number) {
+    const currentAction = reportActions.at(actionIndex);
+    const nextAction = findNextAction(reportActions, actionIndex);
+
+    // Todo first should have avatar - verify that this works with long chats
```

Comment:
Same here, are we doing this in a follow up?
```suggestion
    // Todo first should have avatar - verify that this works with long chats
```

Reply:
+1 lets link issues in which we would handle this in each TODO

Reply:
let's combine these two, because both tough the same part of code - displaying avatars on grouped comments.
Since the current solution is working specifically on InvertedList I had to re-implement it for non-inverted lists.

Both TODO comments are leftovers after this, they should be fixed together